### PR TITLE
Add client_payload

### PR DESCRIPTION
**Pull request by ksgriggs**
- Add client_payload logic

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -59,7 +65,7 @@ function trigger_workflow {
     -H "Accept: application/vnd.github.everest-preview+json" \
     -H "Content-Type: application/json" \
     -H "Authorization: Bearer ${INPUT_GITHUB_TOKEN}" \
-    --data "{\"event_type\": \"${event_type}\", \"client_payload\": {} }"
+    --data "{\"event_type\": \"${event_type}\", \"client_payload\": ${client_payload} }"
```
Comment (wderezin) [inline entrypoint.sh:68]: Do you  need \" around variable like ` \"${client_payload}\" ` ?

Comment (ksgriggs) [reply entrypoint.sh:68]: I can certainly add them and we can test.

Comment (ksgriggs) [reply entrypoint.sh:68]: I just tested this and it doesn't work with \\".  It does work without it.  When we next have a slack call ask me about this.

Comment (ksgriggs) [reply entrypoint.sh:68]: Together we determined that we can expand the variable in the payload if we use

```
${{ github.repository }}
```

Thanks for your help.

**Pull request merged**